lib/core/form/components/widgets/core/form-field.model.ts

```diff
@@ -161,6 +161,7 @@ export class FormFieldModel extends FormWidgetModel {
             this.params = <FormFieldMetadata> json.params || {};
             this.hyperlinkUrl = json.hyperlinkUrl;
             this.displayText = json.displayText;
+            this.isVisible = json.isVisible;
```

Comment:
This is wrong. In the form json coming from APS there is never the isVisible field. We have to evaluate it via the visibility condition